mobile/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -90,9 +90,9 @@
     <string name="tethering_manage_wifi_client_blocked">已隱藏 %1$s：%2$s</string>
     <string name="tethering_manage_wifi_copy_mac">複製 MAC</string>
 
-    <string name="connected_state_incomplete">（正在連線）</string>
-    <string name="connected_state_valid">（已連線）</string>
-    <string name="connected_state_failed">（未連線）</string>
+    <string name="connected_state_incomplete">(正在連線)</string>
+    <string name="connected_state_valid">(已連線)</string>
```

Comment:
Chinese (Traditional) uses full-width punctuation, so it doesn't have to be changed.